app/Services/PlanetService.php

```diff
@@ -1163,7 +1163,7 @@ private function updateResourceProductionStatsInner(Resources $production_total,
      * @return Resources
      * @throws Exception
      */
-    public function getBuildingProduction(string $machine_name, int $building_level = 0): Resources
+    public function getBuildingProduction(string $machine_name, int|null $building_level = null, bool $force_factor = false): Resources
     {
```

Comment:
Could you please update the phpdoc comment to include the new parameter?

Reply:
Oh yes sure my bad

Reply:
Super, thx!